packages/actions/test/utils/authentication.ts

```diff
@@ -497,3 +498,21 @@ export const authenticateUserWithGithub = async (userApp: FirebaseApp, clientId:
     const userFirebaseCredentials = GithubAuthProvider.credential(token)
     return signInToFirebaseWithCredentials(userApp, userFirebaseCredentials)
 }
+
```

Comment:
nit: make only one utility called `setCustomClaims()` which takes a boolean or a custom type. This method should set coordinator or participant to true or false!